third_party/googletest/src/googletest/src/gtest.cc

```diff
@@ -5013,24 +5196,28 @@ class ScopedPrematureExitFile {
             premature_exit_filepath ? premature_exit_filepath : "") {
     // If a path to the premature-exit file is specified...
     if (!premature_exit_filepath_.empty()) {
+#if !GTEST_OS_STARBOARD
```

Comment:
This seems to stub out the `ScopedPrematureExitFile` to quietly do nothing here and below. If it's a useful piece of functionality, we should have a follow-up to implement it.